### ISSUE TITLE
Crash when launching if first subreddit is not frontpage

### ISSUE THREAD

**Issue by Tunous** [ccrama/Slide#103]
## Info
While in loadscreen constant tapping causes crash once 3rd message appears. (Don't ask me why I tapped :p)
## Stacktrace
```java.lang.RuntimeException: Unable to start activity ComponentInfo{me.ccrama.redditslide/me.ccrama.redditslide.Activities.SubredditOverview}: java.lang.NullPointerException
at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2186)
at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2236)
at android.app.ActivityThread.access$600(ActivityThread.java:145)
at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1238)
at android.os.Handler.disp
atchMessage(Handler.java:99)
at android.os.Looper.loop(Looper.java:137)
at android.app.ActivityThread.main(ActivityThread.java:5099)
at java.lang.reflect.Method.invokeN
ative(N
ative Method)
at java.lang.reflect.Method.invoke(Method.java:511)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:803)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:570)
at de.robv.android.xposed.XposedBridge.main(XposedBridge.java:132)
at dalvik.system.N
ativeStart.main(N
ative Method)Caused by: java.lang.NullPointerException
at me.ccrama.redditslide.Activities.SubredditOverview.doSubSidebar(SubredditOverview.java:305)
at me.ccrama.redditslide.Activities.SubredditOverview.setD
ataSet(SubredditOverview.java:1158)
at me.ccrama.redditslide.Activities.SubredditOverview.onCre
ate(SubredditOverview.java:187)
at android.app.Activity.performCre
ate(Activity.java:5117)
at android.app.Instrument
ation.callActivityOnCre
ate(Instrument
ation.java:1081)
at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2150)... 12 more```
Reported via Slide Crash Capture

**Comment by Tunous** [comment]
Actually it seems like that's not because of tapping but the newest version doesn't launch.

**Comment by Tunous** [comment]
Ok I looked into it closely and it is caused by pinned subreddit that appeared before frontpage in overview page. Confirmed this by manually editing shared preferences and removing this subreddit which fixed crash issue.

It might be caused by changes in a5f55db somewhere around line 305 but I don't know codebase yet.

**Comment by ccrama** [comment]
at de.robv.android.xposed.XposedBridge.main(XposedBridge.java:132)

Lots of people are crashing because of xposed, which is very strange, but don't know why this is causing it. Is there any way you could confirm it does not crash with xposed disabled?

**Comment by Tunous** [comment]
#107 is same issue without xposed.

**Comment by ccrama** [comment]
Merged with #107
